lightningd/subd.c

```diff
@@ -469,8 +469,6 @@ static struct io_plan *sd_msg_read(struct io_conn *conn, struct subd *sd)
 		}
 	}
 
-	log_debug(sd->log, "UPDATE %s", sd->msgname(type));
```

Comment:
would it be more appropriate to move this to log at the IO level?

Reply:
The IO level logs only raw messages (hex encoded of course) so far afaik

Reply:
It *can* log messages, but we logged this in lieu of full IO.  We should probably log internal IO messages, but that's a separate change.

Reply:
Does the _internal IO messages_ mean the message between process?